requirements_pre-commit.txt

```diff
@@ -0,0 +1,4 @@
+black==18.9b0; python_version > '3.6'
```

Comment:
Added black to pip requirements, as I guess it's the way every developper installs it (and the way travis does).
Let me know if it's an issue for someone's dev-setup.